### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by mueedc-zz**

### CODE REVIEW

**Comment by mueedc-zz** [comment]
lots of fixes

File: src/keys.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (alaq) [inline src/keys.js]: oh my...

**Review by alaq — CHANGES_REQUESTED** [review]
please fix this

File: src/keys.js (hunk 1 above)
Comment (mueedc-zz) [reply src/keys.js]: yikes

**Review by alaq — APPROVED** [review]

**Pull request closed without merging**